storage/innobase/handler/ha_innodb.cc

```diff
@@ -17540,15 +17544,27 @@ ib_logf(
 	switch(level) {
 	case IB_LOG_LEVEL_INFO:
 		sql_print_information("InnoDB: %s", str);
+#ifdef HAVE_SYSTEMD
+		sd_notifyf(0, "STATUS=InnoDB: Info: %s", str);
+#endif
```

Comment:
I've generally found that if innodb puts out messages after the startup they are normally fairly significant and should be noticed.